frontend/components/Header.tsx

```diff
@@ -7,11 +7,10 @@ import { connect } from 'react-redux';
 import {AppStore} from '../storage/common';
 
 interface HeaderProps{
-	currBreadcrumb: AppStore | undefined;
+	breadcrumb: string;
 }
 const Header: React.FC<HeaderProps> = (props) => {
-	const breadcrumb = props.currBreadcrumb.headerBreadcrumb || 'Missing page name';
-	console.log(breadcrumb);
+	const breadcrumb = props.breadcrumb || 'Missing page name';
```

Comment:
`props.breadcrumb` is entirely under your control, since you create its value in `mapStateToProps`. I think it'd be better to move the fallback `||` down to that function, so that the render logic is as simple as possible. 

As a side note, it's fairly common in React to extract props using ]destructuring assignment](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment):
```
const {breadcrumb} = props;
```

Which has the benefit of neatly scaling with more properties:

```
const breadcrumb = props.breadcrumb;
const permalink = props.permalink;
const somethingElse = props.somethingElse;

// vs

const {breadcrumb, permalink, somethingElse} = props;
```